### PR TITLE
fix(ReportsPartialsCampaigns): change UI with new design

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Apply new design changes.

image

image

### CODE REVIEW

File: src/i18n/en.js [hunk 1]
```diff
@@ -211,6 +211,8 @@ by Doppler and how many don't. By tracking the user's journey you'll be able to
     delivery_rate: `Delivery Rate`,
     emails_delivered: `Emails Delivered:`,
     hard_and_soft: `Hard and Soft bounces`,
+    header_description_shipped: `This is a report of your sent campaign.`,
```
Comment (cbernat) [inline src/i18n/en.js:214]: How about this? @mrunge86 what do you think?
```suggestion
    header_description_shipped: `This is a Report of your Sent Campaign.`,
```

File: src/i18n/en.js [hunk 2]
```diff
@@ -211,6 +211,8 @@ by Doppler and how many don't. By tracking the user's journey you'll be able to
...
+    header_description_shipping: `This is a partial report of your Campaign. Until its completion you can view this data.`,
```
Comment (cbernat) [inline src/i18n/en.js:215]:
```suggestion
    header_description_shipping: `This is a partial Report of your Campaign. Until its completion you can view this data.`,
```

File: src/i18n/es.js [hunk 3]
```diff
@@ -212,6 +212,8 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
     delivery_rate: `Tasa de entrega`,
     emails_delivered: `Emails Entregados:`,
     hard_and_soft: `Rebotes Hard y Soft`,
+    header_description_shipped: `Este es un reporte de tu campaña enviada.`,
```
Comment (cbernat) [inline src/i18n/es.js:215]:
```suggestion
    header_description_shipped: `Este es un Reporte de tu Campaña Enviada.`,
```

File: src/i18n/es.js [hunk 4]
```diff
@@ -212,6 +212,8 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
...
+    header_description_shipping: `Este es un reporte parcial de tu Campaña. Hasta su finalización podrás visualizar estos datos.`,
```
Comment (cbernat) [inline src/i18n/es.js:216]:
```suggestion
    header_description_shipping: `Este es un Reporte parcial de tu Campaña. Hasta su finalización podrás visualizar estos datos.`,
```

File: src/i18n/es.js [hunk 5]
```diff
@@ -212,6 +212,8 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
     delivery_rate: `Tasa de entrega`,
     emails_delivered: `Emails Entregados:`,
     hard_and_soft: `Rebotes Hard y Soft`,
+    header_description_shipped: `Este es un Reporte de tu Campaña enviada.`,
```
Comment (cbernat) [inline src/i18n/es.js:215]:
```suggestion
    header_description_shipped: `Este es un Reporte de tu Campaña Enviada.`,
```
image

File: src/i18n/en.js [hunk 6]
```diff
@@ -211,6 +211,8 @@ by Doppler and how many don't. By tracking the user's journey you'll be able to
     delivery_rate: `Delivery Rate`,
     emails_delivered: `Emails Delivered:`,
     hard_and_soft: `Hard and Soft bounces`,
+    header_description_shipped: `This is a Report of your sent Campaign.`,
```
Comment (cbernat) [inline src/i18n/en.js:214]:
```suggestion
    header_description_shipped: `This is a Report of your Sent Campaign.`,
```

**Comment by cbernat** [comment]
Is there a reason why hard bounces metric does not have thousand separator and the other metrics do?

**Comment by andresmoschini** [comment]
Looks nice!

**Pull request merged**